### ISSUE TITLE
QC Pond-generated assembly Fasta and include metrics in Core data set

### ISSUE THREAD

**Issue by steph-sing** [FDA-ARGOS/data.argosdb#129]

**Comment by steph-sing** [comment]
Dependent on tasks #113 and #114 being completed

**Comment by steph-sing** [comment]
@penningtonea we have the Wuhan fasta ready (please see email). I will move this to Dec and consider it in progress.

**Comment by steph-sing** [comment]
Issue is not relevant. will create a new issue when we are ready to QC Pond teams fasta files.
